mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -1338,6 +1355,31 @@ def update(self, data):
             if self.has_callback:
                 CallbackBlock.create(self.uuid)
             self.__update_pipeline_block()
+        if (
```

Comment:
you can add some of this integration pipeline specific logic in the `IntegrationBlock` class